### PR TITLE
Add a subscribe button to empty feed

### PR DESCRIPTION
**Pull request by Schmiddiii**
# Licensing 

- [x] I confirm that this is either my code or was released under the terms of a GPLv3-or-later compatible license. Also I agree to release it here under the terms of the GPLv3-or-later.

# Description

This adds a "subscribe"-button to an empty feed page.

tubefeeder_subscribe_button

Ping @TheEvilSkeleton if anything can be improved in this regard.

# Linked Issue

Related to #95.

### CODE REVIEW

File: data/resources/ui/feed_page.ui [hunk 1]
```diff
@@ -69,6 +69,22 @@
...
+            <property name="child">
+              <object class="GtkButton" id="btn_add_subscription">
+                <style>
+                  <class name="pill"/>
+                </style>
+                <property name="halign">center</property>
+                <child>
+                  <object class="AdwButtonContent">
+                    <property name="icon-name">list-add-symbolic</property>
+                    <property name="label" translatable="yes">Subscribe to a chanel</property>
```
Comment (TheEvilSkeleton) [inline data/resources/ui/feed_page.ui:82]: I don't think we should be adding a `+` icon. Texts should be fine.

```suggestion
                    <property name="label" translatable="yes">Subscribe to a Channel</property>
```

File: data/resources/ui/feed_page.ui [hunk 2]
```diff
@@ -69,6 +69,22 @@
             <property name="description" translatable="yes">Are you subscribed to channels?</property>
             <property name="icon-name">go-home-symbolic</property>
             <property name="vexpand">True</property>
+
+            <property name="child">
+              <object class="GtkButton" id="btn_add_subscription">
+                <style>
+                  <class name="pill"/>
```
Comment (TheEvilSkeleton) [inline data/resources/ui/feed_page.ui:76]: Suggesting that button (making it blue) would help the user.

```suggestion
                  <class name="pill"/>
                  <class name="suggested-action"/>
```

**Review by TheEvilSkeleton — CHANGES_REQUESTED** [review]
My suggestions aren't final, but I think it's a good start. I'll test once the suggestions are added

**Comment by TheEvilSkeleton** [comment]
That's normal design for GNOME apps that have an empty startup nowadays; Loupe, Bottles, etc. do that

File: data/resources/ui/feed_page.ui [hunk 3]
```diff
@@ -69,6 +69,19 @@
             <property name="description" translatable="yes">Are you subscribed to channels?</property>
```
Comment (TheEvilSkeleton) [inline data/resources/ui/feed_page.ui:69]: We should remove this, as "No Videos Found" implies to subscribe to a channel
```suggestion
```

**Review by TheEvilSkeleton — CHANGES_REQUESTED** [review]

**Comment by Schmiddiii** [comment]
Should we maybe do a similar thing in the filters and subscriptions pages?

**Comment by TheEvilSkeleton** [comment]
Absolutely! I didn't think of that, but yes, that would be great

File: data/resources/ui/filter_page.ui [hunk 4]
```diff
@@ -44,14 +44,25 @@
...
+
+            <property name="child">
+              <object class="GtkButton" id="btn_add_filter">
+                <style>
+                  <class name="pill"/>
+                  <class name="suggested-action"/>
+                </style>
+
+                <property name="halign">center</property>
+                <property name="label" translatable="yes">Add a Filter</property>
```
Comment (TheEvilSkeleton) [inline data/resources/ui/filter_page.ui:63]:
```suggestion
                <property name="label" translatable="yes">Add Filter</property>
```

**Review by TheEvilSkeleton — CHANGES_REQUESTED** [review]

**Comment by TheEvilSkeleton** [comment]
> To be honest, I think the suggestive highlighting of the button is too much.

About that, if you don't like how it looks, then you can always remove the style. You maintain and develop it after all

**Pull request merged**